### PR TITLE
Update SPDX license kind list

### PR DESCRIPTION
**Pull request by aiuto**
Periodic refresh.

### CODE REVIEW

File: tools/BUILD [hunk 1]
```diff
@@ -47,7 +47,7 @@ py_library(
 py_binary(
     name = "write_sbom",
     srcs = ["write_sbom.py"],
-    deps = [":sbom_lib"],
```
Comment (mzeren-vmw) [inline tools/BUILD:50]: The commit description should explain this commented out line. Can it be deleted instead of just commented out?

Comment (aiuto) [reply tools/BUILD]: That was an accident. Reverted.

**Comment by aiuto** [comment]
cc: @weikert

**Comment by aiuto** [comment]
I'ld like to get this in before the 1.0.0 release.

**Review by teeler — APPROVED** [review]

**Review by shs96c — APPROVED** [review]
LGTM

**Pull request merged**